test/TestExpectations.json

```diff
@@ -4372,5 +4366,101 @@
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["cdp", "chrome", "headless"],
     "expectations": ["FAIL", "PASS"]
+  },
+  {
+    "testIdPattern": "[frame.spec] Frame specs Frame Management should send events when frames are manipulated dynamically",
```

Comment:
Intentionally not sorted so we can remove the  `chrome` if after Firefox passes as well.